.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing to Meteor Roles
+
+Any contribution to this repository is highly appreciated!
+
+## Setup development env
+
+### Clone project and create a new branch to work on
+
+First, clone this repository and create a new branch to work on.
+Branch names should follow the GitFlow standard and start with a descriptive prefix of their intended outcome, for example:
+
+- `feature/` for features
+- `fix/` for general fixes
+
+Then the name of the branch should describe the purpose of the branch and potentially reference the issue number it is solving.
+
+```shell
+$ git clone git@github.com:Meteor-Community-Packages/meteor-roles.git
+$ cd meteor-roles
+$ git checkout -b fix-some-issue
```

Comment:
Please change `fix-some-issue` to `fix/some-issue` so it's consistent with the described syntax from above

Reply:
Updated, good find!